### ISSUE TITLE
Payment issues

### ISSUE THREAD

**Issue by MattRaynerDBS** [202-ecommerce/stripe_official#74]
Hi,

We started on version 2.3.1, and had occasional issues with orders not being created after payments had been accepted.

We attempted to upgrade to 2.3.2 and only had more issues. At first, it didn't create the webhook automatically. We managed to eventually get the module to do so after entirely removing it and removing the STRIPE_* configuration keys in the database.

However, now it doesn't capture payments either as the Webhook returns a 'Fatal error' to stripe, this is logged in the stripe_official_processlogger table:
```
Invalid signature : Unable to extract timestamp and signatures from header
```

As far as I can see from the Stripe dashboard, the `STRIPE_WEBHOOK_SIGNATURE` is the correct value.

Unfortunately, even in this case, it still presents an order success page, even though the payment is left uncaptured and no order is created within PrestaShop.

Please advise, as we have had to disable taking Stripe payments at the moment until this is resolved, as we cannot be sure that payments will succeed or even fail in a transparent way to the user.

**Comment by zorrobiwan** [comment]
You should take a look to issue #72 (for the signature pbl - copy paste the right signature manually in the database to make it work) and #71 (for the uncapture pbl with version 2.3.2 - I would suggest you to revert back to 2.3.1)
Good luck

**Comment by mribeiro202** [comment]
Hi @MattRaynerDBS 

We fixed this issue with the version 2.3.6.
Can you make a try and telle me if it works for you ?

have a nice day
